baseplate/clients/kombu.py

```diff
@@ -60,6 +66,53 @@ def exchange_from_config(app_config: config.RawConfig, prefix: str, **kwargs: An
     return Exchange(name=options.exchange_name or "", type=options.exchange_type, **kwargs)
 
 
+class KombuSerializer(abc.ABC):
+    """Interface for wrapping non-built-in serializers for Kombu."""
+
+    @property
+    @abc.abstractmethod
+    def name(self) -> str:
+        pass
+
+    @abc.abstractmethod
+    def serialize(self, obj: Any) -> bytes:
```

Comment:
Note, I want to take a look at generics and see if that is appropriate in some way.